ObjCCBOR/ObjCCBOR.h

```diff
@@ -2,43 +2,11 @@
 //  ObjCCBOR.h
 //  ObjCCBOR
 //
-//  Created by Hamilton Chapman on 20/02/2020.
-//  Copyright © 2020 Ditto. All rights reserved.
+//  Created by Hamilton Chapman on 01/05/2020.
+//  Copyright © 2020 Dash. All rights reserved.
 //
 
 #import <Foundation/Foundation.h>
 
-NS_ASSUME_NONNULL_BEGIN
-
-@interface ObjCCBOR : NSObject
-
-/**
- Decode CBOR data into its corresponding NSObject type. Note that
- this library supports primitives so the resulting type could be any
- of `NSDictionary`, `NSArray`, `NSString`, `NSNumber`, `NSNull` or
- `NSData`.
-
- @param data The data to be decoded.
- @param error An optional error out parameter. If not `NULL`, then
- an error object will be written to this parameter should decoding fail.
- @returns Returns one of `NSDictionary`, `NSArray`, `NSString`,
- `NSNumber`, `NSNull` or `NSData`. Will return `nil` if the decoding
- fails.
- */
-+ (nullable id)decode:(NSData *)data error:(NSError * _Nullable __autoreleasing *)error;
-
-/**
- Encodes an NSObject into its CBOR representation. Note that `object`
- may only be one of `NSDictionary`, `NSArray`, `NSString`, `NSNumber`,
- `NSNull`, `NSData` or their mutable variants where appropriate.
-
- @param object The object to be encoded.
- @param error An optional error out parameter. If not `NULL`, then
- an error object will be written to this parameter should encoding fail.
- @returns Returns the encoded data or `nil` if the encoding fails.
- */
-+ (NSData *)encode:(NSObject *)object error:(NSError * _Nullable __autoreleasing *)error;
-
-@end
-
-NS_ASSUME_NONNULL_END
+#import "CBOR.h"
+#import "CBORRepresentable.h"
```

Comment:
In the umbrella header, these should use angle bracket syntax.

```suggestion
#import <ObjCCBOR/CBOR.h>
#import <ObjCCBOR/CBORRepresentable.h>
```

---

As an aside: choosing the framework header over simply including everything in there was definitely the right move.

> Including framework header files. Each framework should have a header file, named after the framework, that includes all the public header files of the framework.
> 
> Header file | Framework
> -- | --
> Foundation.h | Foundation.framework.
> 

[Coding Guidelines for Cocoa](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingBasics.html#//apple_ref/doc/uid/20001281-1002806-BBCFIFEH)